aim/cli/manager/manager.py

```diff
@@ -45,6 +45,7 @@ def run_up(args):
         'host': 'http://' + args['--host']
     }
 
+    time.sleep(2)
```

Comment:
why the sleep was moved here?

Reply:
It need to sleep a while before the server will be live, and no need to read `stderr` in that time